.gitignore

```diff
@@ -57,3 +57,5 @@ default.profraw
 # Everything without a .enc extension is ignored
 .configure-files/*
 !.configure-files/*.enc
+
+Gemfile.lock
```

Comment:
Wasn't sure if we needed to or wanted to have the gem file/gemlock file in git.  Seemed odd to me to track it, but happy to undo this change if people disagree

Reply:
@charliescheer Yep, we track `Gemfile.lock` in all the apps so the dependency versions stay consistent.

Reply:
Okay, I removed the gem lock file from the gitignore